pkg/destroy/aws/aws.go

```diff
@@ -1712,7 +1723,13 @@ func deleteRoute53(session *session.Session, arn arn.ARN, logger logrus.FieldLog
 		return errors.Errorf("unrecognized Route 53 resource type %s", resourceType)
 	}
 
-	client := route53.New(session)
+	awsConfig := aws.NewConfig()
+	switch *session.Config.Region {
+	case endpoints.CnNorth1RegionID, endpoints.CnNorthwest1RegionID:
+		awsConfig.WithRegion(endpoints.CnNorthwest1RegionID)
+		awsConfig.WithEndpoint("https://route53.amazonaws.com.cn")
```

Comment:
i think this needs assignement to be more readable.

Reply:
done, added some notes